src/matrix/room/timeline/persistence/GapWriter.js

```diff
@@ -163,7 +163,7 @@ export class GapWriter {
         if (!Array.isArray(chunk)) {
             throw new Error("Invalid chunk in response");
         }
-        if (typeof end !== "string") {
+        if (typeof end !== "string" && typeof end !== "undefined") {
```

Comment:
This was already added on master, you can just merge it in now.

Reply:
This PR is on `thirdroom/dev` so I was wondering is it safe to merge master?